### DISCUSSION TITLE
restart explorer

### DISCUSSION THREAD

**Discussion by Alhlhli** [moudey/Shell#295] [Q&A] [answered]
I've been trying to add the Restart Explorer menu to the taskbar.nss  list

I tried with cmd
 command
 I dont sucss

Orders not executed

		item(title='Restart Explorer' image=\uE092 cmd='taskkill /f /im explorer.exe  & start explorer.exe')
		item(title='Restart Explorer' image=\uE092 command.restart_explorer)

**Comment by moudey** [comment, marked as answer]
اهلا امير 
الامر الثاني ينقصه فقط اضافة خاصية `cmd` ليصبح بهذا الشكل
```
item(title='Restart Explorer' image=\uE092 cmd=command.restart_explorer)
```

**Reply by Alhlhli** [reply]
Thank you very much 
million Thanks

**Reply by kaser2010** [reply]
المشكلة لو عملت شي خطأ بالاكواد وتريد تعمل ريستارت  explorer.exe ماتقدر لأن البرنامج shell ماراح يشتغل معاك
لكن عملت طريقة حلوه يكون الامر في This PC يمين بالماوس ويكون فيه Restart Windows Explorer
تحفظ الامر Notepad والامتداد .reg
وبعدين افحه وبس
```
Windows Registry Editor Version 5.00

[HKEY_CLASSES_ROOT\CLSID\{20D04FE0-3AEA-1069-A2D8-08002B30309D}\shell\Restart Windows Explorer]
"MUIVerb"="Restart Windows Explorer"
"Icon"=hex(2):25,00,50,00,72,00,6f,00,67,00,72,00,61,00,6d,00,46,00,69,00,6c,\
  00,65,00,73,00,25,00,5c,00,45,00,61,00,73,00,79,00,20,00,43,00,6f,00,6e,00,\
  74,00,65,00,78,00,74,00,20,00,4d,00,65,00,6e,00,75,00,5c,00,45,00,63,00,4d,\
  00,65,00,6e,00,75,00,2e,00,65,00,78,00,65,00,2c,00,32,00,33,00,00,00

[HKEY_CLASSES_ROOT\CLSID\{20D04FE0-3AEA-1069-A2D8-08002B30309D}\shell\Restart Windows Explorer\command]
@=hex(2):22,00,25,00,50,00,72,00,6f,00,67,00,72,00,61,00,6d,00,46,00,69,00,6c,\
  00,65,00,73,00,25,00,5c,00,45,00,61,00,73,00,79,00,20,00,43,00,6f,00,6e,00,\
  74,00,65,00,78,00,74,00,20,00,4d,00,65,00,6e,00,75,00,5c,00,45,00,63,00,4d,\
  00,65,00,6e,00,75,00,2e,00,65,00,78,00,65,00,22,00,20,00,2f,00,52,00,65,00,\
  45,00,78,00,70,00,6c,00,6f,00,72,00,65,00,72,00,00,00
```

**Reply by moudey** [reply]
@kaser2010
سوف اقوم بتزويد شيل باختصار يتيح لك إعادة تشغيل المستكشف من خلال لوحة المفاتيح حتى يسهل على المستخدم اعادة تحميل الاعدادات في حالة تحريرها.

**Reply by Alhlhli** [reply]
ملف الريجستري لم يتم تشغيله ويعطي رسالة خطأ 

لدي ويندوز 11

**Reply by kaser2010** [reply]
استعمل هذا البرنامج وفيه المطلوب
`https://www.sordum.org/7615/easy-context-menu-v1-6/`

**Comment by Alhlhli** [comment]
I found the following command in taskbar.nss 
	item(vis=@key.shift() title=title.exit_explorer cmd=command.restart_explorer)
